bridges/zap/zap_encoder.go

```diff
@@ -0,0 +1,211 @@
+package zap
+
+import (
+	"encoding/json"
+	"fmt"
+	"strconv"
+	"time"
+
+	"go.opentelemetry.io/otel/log"
+	"go.uber.org/zap/zapcore"
+)
+
+// not optimized yet
+// this file implements object and array encoder - similar to memory encoder by zapcore
+var (
+	_ zapcore.ObjectEncoder = (*OtelObjectEncoder)(nil)
+	_ zapcore.ArrayEncoder  = (*sliceArrayEncoder)(nil)
+)
+
+type OtelObjectEncoder struct {
```

Comment:
The short hand for OpenTelemetry is OTel.

```suggestion
type OTelObjectEncoder struct {
```

However, with the above package name suggestion, there is little need to include `OTel` in the name at all given it will be in the `otelzap` package scope.

```suggestion
type ObjectEncoder struct {
```

Would be better.